cookbook/core/flyte_basics/lp.py

```diff
@@ -87,3 +97,62 @@ def go_greet(day_of_week: str, number: int, am: bool = False) -> str:
     else:
         # We're extra enthusiastic on weekends
         print(morning_greeting(number=3, day_of_week=weekday))
+
+# %%
+# Partial Inputs for Launch Plans
+# ###############################
+# Launch plans bind a partial or complete list of inputs necessary to launch a workflow. Launch plan inputs must only assign inputs already defined in the reference workflow definition.  Refer to to :ref:`primary launch plan concept documentation <concepts-launchplans-inputs>` for a detailed introduction to launch plan input types.
+#
+# For example, for the following workflow definition:
+
+.. code:: python
+
+    from datetime import datetime
+    from flytekit import workflow
+
+    @workflow
+    def MyWorkflow(region: str, run_date: datetime, sample_size: int=100):
+        ...
```

Comment:
Isn't it better if we have the workflow defined?

Reply:
@cosmicBboy ?